activerecord/lib/active_record/connection_adapters/postgresql/schema_creation.rb

```diff
@@ -9,6 +9,13 @@ def visit_AlterTable(o)
             super << o.constraint_validations.map { |fk| visit_ValidateConstraint fk }.join(" ")
           end
 
+          def visit_DropTableDefinition(o)
+            sql = "DROP TABLE "
+            sql += "IF EXISTS " if o.if_exists
+            sql + quote_table_name(o.name)
```

Comment:
```suggestion
            sql += quote_table_name(o.name)
```
or `<<` should be added here right?